app/models/cart.rb

```diff
@@ -14,10 +18,8 @@ def add_beer(beer, quantity)
     end
   end
   
-  def total # use reduce
-    total = 0
-    contents.each { |k, v| total += v["beer_price"].to_i }
-    total
+  def total
+    contents.values.reduce(0) { |sum, value| sum += value["beer_price"].to_i }
```

Comment:
Useless assignment to variable - `sum`. Use just operator `+`.<br>Prefer `inject` over `reduce`.